### PR TITLE
Add feature page for network resilience features

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

**Comment by brian-avery** [comment]
@istio/wg-networking-maintainers can you please review?

**Comment by johnma14** [comment]
@howardjohn @istio/wg-networking-maintainers can we get a review on this one. Thank you.

File: features/network-resiience.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+- [ ] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [ ] Automated integration tests cover core use cases with the feature enabled. 
+- [ ] When disabled, the feature does not affect system stability or performance. 
+
+**API**
+
+- [ ] Initial API review.
```
Comment (howardjohn) [inline features/network-resiience.md:121]: if this is stable shouldn't these all be checked?

Comment (johnma14) [reply features/network-resiience.md]: Done.

**Pull request closed without merging**